src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const NavBar: React.FC = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => (
+            classNames('navbar-item ', {
+              'is-active': isActive,
+            })
+          )}
+        >
+          Home
+        </NavLink>
+        <NavLink
+          to="/tabs"
+          className={({ isActive }) => (
+            classNames('navbar-item', {
+              'is-active': isActive,
+            })
+          )}
+        >
+          Tabs
+        </NavLink>
```

Comment:
You could create a separate component for this link and reuse it, but it's not critical